CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Welcome and thank you for wanting to contribute! 
 
-## In a hurry? Here's the short version:
+## Get started
```

Comment:
How about mentioning Weekly Updates here? Something along the lines of:

- Want to get the latest updates? Check the [WEEKLY_UPDATES.md](https://github.com/opentofu/opentofu/blob/main/WEEKLY_UPDATES.md) 

Bonus points if you folks want to commit to the weekly DEV gatherings on Wednesdays and mention it here with the calendar invite 😅 and maybe mention the TSC (which is mentioned at the end of the document).

Reply:
Good point. If the Wednesday meetings become regular, we could add them here.